dsrc/main.lit

```diff
@@ -54,6 +55,21 @@ bool noCompCmd = true;
 string outDir = "."; // Default is current directory
 ---
 
+This program uses a number of block modifiers in order to facilitate certain functionality.
+i.e. If you don't wish a code block to be woven into the final HTML then the `no-weave` 
+modifier will indicate this for you.
+
+Each modifier is represented by this list of enums:
+
+--- Modifiers
+enum Modifier {
+        noWeave,
+        noTangle,
+        additive, //+=
+        coleq // :=
```

Comment:
I wasn't sure what to call it at the time. I am not sure if additive is the right name ether. Tell me if you have a more descriptive version for any of these.